docs/user/AggFunc.md

```diff
@@ -0,0 +1,112 @@
+# Aggregate Function
+Aggregate functions output a single result value from a collection of input values.
+
+Unlike SQL, `GROUP BY` is not required to use the aggregate function. 
+
+Unless otherwise specified, aggregate functions will skip `NULL` and `MISSING`
+
+## AVG
+
+Returns the average (arithmetic mean) of the input expression values.
+
+Signature
+: `AVG: COLLECTION OF NUMERIC TYPE -> DECIMAL OR NULL`
+
+Header
+: `AVG(expr)`
+
+Purpose
+: Given a collection of numeric type values, evaluates the arithmetic mean of the input expression values. 
+
+Example
+: 
+
+```SQL
+    AVG(<< 1,2,3 >>) -- 2 (Decimal type with Precision of 1 and Scale of 0)
+    AVG(<< 1,2,3,'1'>>) --error
+    AVG(<<MISSING>>) -- NULL
+    AVG(<< 1,2,3, MISSING >>) -- 2
+```
+
+## COUNT
+
+Returns the number of “rows” in the input.
+
+Signature
+: `COUNT: STAR -> INT8`
+: `COUNT: Any DATATYPE -> INT8`
+
+Header
+: `COUNT(*)`
+: `COUNT(expr)`
+
+Purpose
+: Given a collection of variable in any date type, calculate the number of value in the collection.
+Note: `COUNT(expr)` will skip the null and missing value, but COUNT(*) will not. 
+
+Example
+: 
+
+```sql
+COUNT(<< 1,2,3 >>) -- 3
+COUNT(<< 1,2,3,<<'1', {'a':1} >> >>) -- 4
+-- COUNT(expr) ignores null and missing
+SELECT COUNT(a) FROM <<MISSING>> AS a -- 1 
+SELECT COUNT(a) FROM <<MISSING, 1, 2>> AS a -- 2 
+
+-- COUNT(*) includes null and missing
+SELECT COUNT(*) FROM <<MISSING>> AS a -- 1 
+SELECT COUNT(*) FROM <<MISSING, 1, 2>> AS a -- 3 
+```
+
+## MIN/MAX
+
+Returns the MIN/MAX value of the input expression values.
```

Comment:
Include information regarding the comparator. Is the comparator consistent with `ORDER BY`?

Reply:
Addressed in the new commit. The comparator is consistent with `ORDER BY` with one exception, it skips `NULL` and `MISSING`.